guides/common/modules/con_managing-multiple-content-view-environments.adoc

```diff
@@ -0,0 +1,10 @@
+[id="managing-multiple-content-view-environments"]
+= Managing multiple content view environments
+
+You can manage multiple content views in different environments by organizing, promoting, and maintaining them effectively.
+
+You can assign multiple content view environments to hosts from the server using Hammer or from the host using Subscription Manager. 
+You can also view content view environment assignments, understand environment ordering and priority, and compare multiple content view environments with composite content views.
```

Comment:
The second and third items here are not tasks that users accomplish; they are just topics covered later that one may need to understand. 

May want to delete those and instead highlight that viewing also works on web UI: "You can view content view environment assignments both in Hammer and web UI"